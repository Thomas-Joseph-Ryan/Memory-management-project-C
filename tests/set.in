Set a 1 2 3
set b 345 3 4 5
SET c 1 
set thejustrightkey 1
set keythatistoolong 1
set 1 1
set a 2 3 4
set d d
set e a
set f 1a 1 2 3
set f a!
set g h
set h ! +
