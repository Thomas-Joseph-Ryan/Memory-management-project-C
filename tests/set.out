> ok

> ok

> ok

> ok

> key is not permitted

> key is not permitted

> ok

> not permitted

> ok

> invalid Command

> no such key

> no such key

> invalid Command

> 
bye
